Modules/RosterManager/RosterConfiguration.lua

```diff
@@ -169,26 +169,20 @@ local TRANSFORMS = {
 }
 
 function RosterConfiguration:inflate(data)
-    for i, key in ipairs(self:fields()) do
+    for i, key in ipairs(fields()) do
         self._[key] = TRANSFORMS[key](data[i])
     end
 end
 
-function RosterConfiguration:deflate()
-    local result = {}
-    for _, key in ipairs(self:fields()) do
-        table.insert(result, self._[key])
+local function validate(self, option, value)
+    local callback = "_validate_" .. option
+    if type(self[callback]) == "function" then
+        local r = self[callback](value)
+        return r
     end
 
-    return result
-end
-
-function RosterConfiguration:Copy(o)
```

Comment:
[Modules/AuctionManager/GUI.lua#L290](https://github.com/ClassicLootManager/ClassicLootManager/blob/master/Modules/AuctionManager/GUI.lua#L290)

Reply:
Restored it.